composer.json

```diff
@@ -65,7 +65,7 @@
             "bin/phpunit tests/Integration/"
         ],
         "run-tests-with-clover": [
-            "bin/phpunit tests/Unit/ --coverage-clover build/logs/clover.xml"
+            "XDEBUG_MODE=coverage bin/phpunit tests/Unit/ --coverage-clover build/logs/clover.xml"
```

Comment:
We can safely set this globally under `env.global`.

Reply:
makes sense, and is much more elegant